lib/buttons.ex

```diff
@@ -0,0 +1,136 @@
+defmodule Inky.Buttons do
+  @moduledoc """
+  The Inky Impression has 4 buttons that are monitored independently from the display
+  and can be started in supervison.
+
+  Supply the `:handler` option as either a pid, or `{module, function, args}` tuple
```

Comment:
Would it make sense to also support an atom name for a process? I think that'd be convenient considering these are all likely singletons.

Reply:
@lawik I've added support for atom name in https://github.com/pappersverk/inky/pull/48/commits/40cda8453026ba9822f60912ec9172f477c526f3